README.md

```diff
@@ -1,27 +1,163 @@
+
+**Glossary**
+
+- VEP
+- EF: Enhancement Freeze
+- CF: Code Freeze
+- RC: Release Candidate
+- Alpha/Beta
+- release team
+
+## Process (DRAFT 2nd iteration)
+
+1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. 
+   See the [VEP template](https://github.com/kubevirt/enhancements/blob/main/veps/NNNN-vep-template/vep.md), 
+   each section needs to be filled even if the section doesn't apply to the VEP.
+
+2. **Visibility and Tracking**: The Author of VEP will open an issue to track their progress, maturity stages (alpha,
+   beta, GA), list the associated bugs, and user feedback.
+   See the [VEP issue template](https://github.com/kubevirt/enhancements/blob/main/.github/ISSUE_TEMPLATE/vep.md), 
+   each section should be filled.
+
+3. **SIG Review and Collaboration**: Each VEP will have one target SIG. In case of a cross SIG feature, the most relevant 
+   SIG should be chosen. The SIG will decide if the VEP is accepted and whether it should be tracked for the upcoming release. 
+   If so, the SIG will assign dedicated reviewers to oversee the proposal, collaborate with others SIGs as needed, and provide 
+   feedback or veto when necessary. The author and reviewers work towards merged VEP.
+
+   Each SIG (Compute, Network, Storage) should sign-off on the VEP with an LGTM from a SIG representative (chair or approver), 
+   even if the proposal doesn't affect them. The sign-off works here as acknowledgment of the changes.
+
+4. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the
+   project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community
+   support and a commitment to implementation.
+
+    - On the EF, approved VEPs will be announced to kubevirt-dev mailing list in order to draw attention to them.
+    - Each week a release team will check tracked VEP in order to assure activity.
+
+5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with
+   the Kubernetes KEP process. It will ensure that each enhancement includes all the relevant information, including the
+   design and the state.
+
+### Responsibilities
+
+#### VEP Owner
 
 The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated).
+In addition, they are encouraged:
+
+- talk about the design in the unconference sessions to bring everyone on the same page about the problem, 
+  use-cases/design etc...
+- join the relevant sig calls for further discussions, and seeking for reviews.
+- ask for reviews on community calls if this is a cross-sig VEP.
+
+#### SIGs
+
+The responsibility of the SIGs is to make sure the VEP is implemented, not diverging and that the implementation 
+is not lacking behind the VEP, following this non-exhaustive list meant as checklist:
```

Comment:
To emphasize that the SIG is not responsible that the VEP is implemented but is committed to help the VEP owner to do so.

```suggestion
The responsibility of the SIGs is to do their best to help ensure the VEP is implemented, not diverging and that the implementation 
is not lacking behind the VEP, following this non-exhaustive list meant as checklist:
```